fp-andrewMorrison-peterChristakos-julianPinzer-katherineThompson

andrewrm98 opened the pull request:
https://fp-admin-page.glitch.me

andrewrm98 committed:
Project description

jpinz committed:
Initial Commit

Commit:
added connection to mysql

charlieroberts commented:
This sounds good! So you'll have an interface that enables users to navigate between different tables and change data however they want? Will this work with any MySQL database they provide credentials for?

jpinz committed:
Changed over to create-react-app

andrewrm98 commented:
This is for a specific financials MySQL database that we are creating. The user will be able to edit the tables however they want and we will have D3 components to visualize the data that is entered.

jpinz committed:
Made MYSQL work, gets a list of users

Currently using an express server and react server with proxying, trying to figure out if there's a way around it.

jpinz committed:
Added react-router

jpinz committed:
Added fake data to populate table and chart

jpinz committed:
Changed actual to approved to keep in sync with MySQL

jpinz committed:
Tried adding the ability to add, remove, and edit budgets.

It's weird though because I'm writing it for an object, not with MySQL Queries, so that will have to be fixed.

andrewrm98 committed:
Added username, password fields and submit button

Commit:
Added basic tabulator framework

jpinz committed:
Fixed data for Peter

Commit:
added bulma theme to table

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

andrewrm98 committed:
Added styling to login page (bulma and Awesome Fonts added)

andrewrm98 committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

andrewrm98 committed:
Password field hidden

jpinz committed:
Changed tabulator to react-tabulator

Commit:
reorganized dashboard framework and added submission fields

Commit:
added submit button

jpinz committed:
Tabulator uses bulma now

Commit:
Dashboard front end done

Commit:
cleaned up dashboard front end

andrewrm98 committed:
Better header

jpinz committed:
Trying to add logins

jpinz committed:
forgot to include package-lock

jpinz committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
restructured dashboard layout, submit button works

jpinz committed:
Trying to add login functionality

jpinz committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

jpinz committed:
Made an online mysql server

Hope this works!

jpinz committed:
It all works with the online MySQL server now.

Add and Delete work. Trying to fix the editing functionality

Commit:
continuation of graph ish

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

 Conflicts:
	public/index.html
	src/views/Dashboard/Dashboard.js
	src/views/Home/Home.js

Commit:
added alerts, changed title

andrewrm98 committed:
Login page fixed and header changed a bit

andrewrm98 committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
got rid of export buttons

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

andrewrm98 committed:
Styling

andrewrm98 committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
simple graph showing

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

 Conflicts:
	package-lock.json
	package.json

andrewrm98 committed:
Numbers only for amount fields

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
Login / submit disables

andrewrm98 committed:
Fixed update

Commit:
more graph stuff

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
readme draft

Commit:
our graph data (still need to fix aspect ratio)

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
graph class css

jpinz committed:
Trying to get login working

jpinz committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

Commit:
larger graph w/ legend

jpinz committed:
Login works, but state isn't maintained

jpinz committed:
Merge branch 'master' of https://github.com/andrewrm98/fp

jpinz committed:
Update README.md

jpinz committed:
Update README.md

Added link to the readme

Commit:
cleaned up README

Commit:
Merge branch 'master' of https://github.com/andrewrm98/fp

 Conflicts:
	README.md
